src/documents/tutorial-creatingAPreferencesEditorUsingThePreferencesFramework/CreatingEnactors.md

```diff
@@ -123,7 +123,7 @@ fluid.defaults("gpii.pmt.enactors.magnification", {
     ....
     modelListeners: {
         "magnificationFactor": {
-            funcName: "gpii.pmt.enactors.magnification.magnify",
+            listener: "gpii.pmt.enactors.magnification.magnify",
```

Comment:
According to the conversation with Bosmon (https://botbot.me/freenode/fluid-work/2015-03-10/?tz=America/Havana, 1:30PM), both "funcName" and "listener" are currently valid and equivalent. Since the plan is to gradually eliminate "listener", I will replace "listener" here back to "funcName", as well as other places in docs.